src/core/PluginFactory.cpp

```diff
@@ -86,7 +86,7 @@ void PluginFactory::setupSearchPaths()
 #endif
 	// Or via an environment variable:
 	QString env_path;
```

Comment:
```suggestion
```
Looks like this QString can be removed.

Reply:
Oh, I should've assigned it to that variable. If its unused I'll try to remove it soon. Thanks for the heads up.